lib/src/screens/task_list.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:flutter/material.dart';
+
+class TaskListScreen extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      key: Key('task_list_screen'),
+      appBar: AppBar(title: Text('Task List')),
+      body: BodyLayout(),
+    );
+  }
+}
+
+class BodyLayout extends StatelessWidget {
+  List<String> titles = [
```

Comment:
Maybe add TODO comments above the `title` and `subtitles` lists here like: 
`TODO: Replace after creating a Task class`